runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -108,18 +109,15 @@ pub fn measure_actions(
 }
 
 // TODO: super-ugly, can achieve the same via higher-level wrappers over POSIX read().
-#[cfg(any(target_arch = "x86_64"))]
+#[cfg(target_family = "unix")]
```

Comment:
Please make this a PR on its own and let's commit it independently of migration to stable Rust, as it is valuable anyway.

Reply:
Do you mean just this line or the change to use file read?